completions/tar

```diff
@@ -437,10 +444,18 @@ _comp_cmd_tar__cleanup_prev()
     fi
 }
 
+_comp_cmd_tar__is_bsdtar()
```

Comment:
I'm not sure about this, does this make sense as a function?
If not, I'm not sure where to define it as a variable

Reply:
I think it's fine as a function. Its contents could be reduced to just this though:
```
[[ ${COMP_WORDS[0]} == ?(*/)bsdtar ]]
```